src/components/Explorer/ExplorerPage.vue

```diff
@@ -0,0 +1,200 @@
+                      <el-date-picker
+                        style="width: 100%"
+                        v-model="dateFrom"
+                        type="datetime"
+                        placeholder="Date from"
+                      />
+                    </el-form-item>
+                  </el-form>
+                </el-col>
+                <el-col :span="12" :offset="1">
+                  <el-form style="width: 100%">
+                    <el-form-item label="Date to">
+                      <el-date-picker
+                        style="width: 100%"
+                        v-model="dateTo"
+                        type="datetime"
+                        placeholder="Date to"
+                      />
+                    </el-form-item>
+                  </el-form>
+                </el-col>
+              </el-row>
+            </div>
+            <el-table
+              class="transactions_table"
+              :data="transactions">
+              <el-table-column label="Time" min-width="175" prop="createdTime" sortable>
+                <template slot-scope="scope">
+                  {{ formatDateLong(scope.row.createdTime) }}
+                </template>
+              </el-table-column>
+              <el-table-column label="From" sortable>
+                <template slot-scope="scope">
+                  {{ scope.row.srcAccountId }}
+                </template>
+              </el-table-column>
+              <el-table-column label="To" prop="destAccountId" sortable>
+              </el-table-column>
+              <el-table-column label="Amount" prop="amount" sortable>
+              </el-table-column>
+              <el-table-column label="Asset" prop="assetId" sortable>
+                <template slot-scope="scope">
+                  {{ assetName(scope.row.assetId) }}
+                </template>
+              </el-table-column>
+              <el-table-column label="Description" prop="description">
+              </el-table-column>
+            </el-table>
+          </el-card>
+        </el-col>
+      </el-row>
+    </el-main>
+  </el-container>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex'
+import dateFormat from '@/components/mixins/dateFormat'
+import currencySymbol from '@/components/mixins/currencySymbol'
+
+const BLOCK_TYPE = 'Block'
+const TRANSACTION_TYPE = 'Transaction'
+const ACCOUNT_TYPE = 'Account'
+
+export default {
+  name: 'explorer-page',
+  mixins: [
+    dateFormat,
+    currencySymbol
+  ],
+  data () {
+    return {
+      searchField: '',
+      searchType: [BLOCK_TYPE, TRANSACTION_TYPE, ACCOUNT_TYPE],
+      currentSearchType: ACCOUNT_TYPE,
+      dateFrom: '',
+      dateTo: ''
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'searchedTransactions'
+    ]),
+    transactions () {
+      let transactions = [...this.searchedTransactions]
+
+      if (this.dateFrom > 0) {
+        transactions = transactions.filter(item => item.createdTime > +this.dateFrom)
```

Comment:
Bad practice to use unary plus before date - https://stackoverflow.com/questions/14255664/new-date-is-this-good-practice

129 and 133 lines